### ISSUE TITLE
slide-animations are not the same

### ISSUE THREAD

**Issue by wburgers** [PolymerElements/neon-animation#106]
The slide-down animation is the only one that ends at none.
The rest of the slide animations (up, left and right) start at none.

I think it is better if they all have the same consistent behavior.

On another note, when using a cascaded-animation to apply a slide animation, I can't seem to set the transformOrigin property.

```javascript
             {
                name: 'cascaded-animation',
                animation: 'slide-up-animation',
                nodedelay: 0,
                transformOrigin: '100% 0',
                timing: {
                  delay: 200
                }
              }
```

**Comment by addyosmani** [comment]
Without knowing the history here, I otherwise think consistency in `animation` behaviour makes sense but this might require some further digging. @blasten are you currently maintaining `neon-animation`?

**Comment by ianlet** [comment]
About the slide down animation, I noticed this problem and fixed it in PR #82. This PR is also about consistency problems. At the moment, the element offers animations to slide from left and slide from right, but none for slide from up and slide from down. So added I fixed it with my PR.

**Comment by wburgers** [comment]
@ianlet Ah cool, I looked through the issue list, but didn't find anything there.
Your PR looks good, hope the maintainers merge it soon :+1:

**Comment by ianlet** [comment]
@wburgers my bad, I should have opened an issue to state the slide down animation problem.

**Comment by cdata** [comment]
Thanks for reporting and for the contribution. I will take a look.

**Comment by wburgers** [comment]
fixed in bb65482cf96ff7fd4c79d9c2f448d046f1ee51a0
